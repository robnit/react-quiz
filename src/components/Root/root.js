import React from 'react'
import App from '../App/App'


const Root = () => (
  <App />
)

export default Root