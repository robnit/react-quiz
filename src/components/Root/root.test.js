import React from 'react'
import Root from './Root'
import { shallow } from 'enzyme'


describe('root', () => {
  
  it('should match snapshots', () => {
    const wrapper = shallow(<Root/>)

    expect(wrapper).toMatchSnapshot()
  })
})
