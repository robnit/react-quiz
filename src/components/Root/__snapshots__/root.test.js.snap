// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`root should match snapshots 1`] = `<App />`;
